src/node/node.js

```diff
@@ -41,6 +41,7 @@ const DEFAULT_OPTIONS = {
     onReady: (node) => {},
     onPeerConnected: (peer) => {},
     onPeerRemoved: (peer) => {},
+    isAnonymous: false
```

Comment:
This is just to display it in the API, right? In that case, we can add it in the API, not here.

Reply:
Yeah that's right, just wantet to make sure all the objects are actually the same. But i can add it in the API only.